package.json

```diff
@@ -13,10 +13,10 @@
     "@quasar/extras": "^1.0.0",
     "@vuex-orm/core": "^0.36.3",
     "@vuex-orm/plugin-search": "^0.24.1",
-    "axios": "^0.18.1",
+    "axios": "^0.21.1",
```

Comment:
Whenever we want to bump up a version, we need to be super careful because when working in a team it might break for others.